rf_tests.go

```diff
@@ -362,6 +369,108 @@ func uploadRFML(c cliContext, api rfmlAPI) error {
 	if err != nil {
 		return cli.NewExitError(err.Error(), 1)
 	}
+
+	wisp_tests, err := readWispFiles([]string{c.String("test-folder")})
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+	err = uploadWispFiles(wisp_tests, false, api)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	return nil
+}
+
+// uploadSingleWispFile uploads wisp file syntax by
+// trying to parse the file and sending any parse errors to the caller
+func uploadSingleWispFile(filePath string) error {
+	wisp, _ := readWispFile(filePath)
```

Comment:
Why ignore errors here? Possibly it's safe because if there was an error than `wisp` will have be an empty struct with `TestID = 0`, and this will get caught by `UpdateWisp` - but we should fail quicker?

Reply:
Yes, that is why I decided to ignore the error here. I wanted to have one less `if err != nil { . . . }` in the function, but failing quicker makes sense, plus you'll also get a more descriptive error to the actual issue.